controllers/cache/labels.go

```diff
@@ -0,0 +1,40 @@
+package cache
+
+const (
+	AppLabel = "harbor-cluster-operator"
+)
+
+// NewLabels returns new labels
+func (redis *RedisReconciler) NewLabels() map[string]string {
+	dynLabels := map[string]string{
+		AppLabel:                     redis.Name,
+		"app.kubernetes.io/name":     "cache",
+		"app.kubernetes.io/instance": redis.Namespace,
+		"goharbor.io/harbor-cluster": redis.Name,
```

Comment:
The value of the label `"goharbor.io/harbor-cluster"` should be the name of the cluster, not the name of component service?

Reply:
this name is the name of the cluster
```golang
return &cache.RedisReconciler{
		CXT:           ctx,
		HarborCluster: harborCluster,
		Client:        options.Client,
		Recorder:      options.Recorder,
		Log:           options.Log,
		DClient:       options.DClient,
		Scheme:        options.Scheme,
		Properties:    &lcm.Properties{},
		Name:          harborCluster.Name,
		Namespace:     harborCluster.Namespace,
	}
```